routes/routes.go

```diff
@@ -2,13 +2,23 @@ package routes
 
 import (
 	"github.com/labstack/echo/v4"
+	"github.com/DSSD-Madison/gmu/routes/api"
```

Comment:
in golang, we organize imports like this:

system

external

internal

i'm not going to add a comment for all the places you need to fix this so look through your files